### PR TITLE
Upadated README.md; Hacktoberfest-2018

### PR DESCRIPTION
**Pull request by preeti98**
Please do let me know if I need to make any further specific changes. Thank you!
This is a PR in response to issue #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Flow Network [![License](http://img.shields.io/badge/license-MIT-lightgrey.svg?style=flat)][License] [![Flattr this](http://img.shields.io/badge/flattr-donate-lightgrey.svg?style=flat)][Donate] [![Build Status](http://img.shields.io/travis/flow/network/develop.svg?style=flat)](https://travis-ci.org/flow/network) [![Coverage Status](http://img.shields.io/coveralls/flow/network/develop.svg?style=flat)](https://coveralls.io/r/flow/network)
+# Flow Network [![License](http://img.shields.io/badge/license-MIT-lightgrey.svg?style=flat)][License] [![Flattr this](http://img.shields.io/badge/flattr-donate-lightgrey.svg?style=flat)][Donate] [![Build Status](https://circleci.com/gh/GlowstoneMC/network)](https://travis-ci.org/flow/network) [![Coverage Status](http://img.shields.io/coveralls/flow/network/develop.svg?style=flat)](https://coveralls.io/r/flow/network)
```
Comment (aramperes) [inline README.md]: Remove the `[Flattr this][Donate]` (donate shield)

File: README.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Are you a talented programmer looking to contribute some code? We'd love the hel
 * Please follow the above guidelines for your pull request(s) accepted.
 * For help setting up the project, keep reading!
 
-Love the project? Feel free to [donate] to help continue development! Flow projects are open-source and powered by community members, like yourself. Without you, we wouldn't be here today!
+Love the project? Feel free to donate to help continue development! Flow projects are open-source and powered by community members, like yourself. Without you, we wouldn't be here today!
```
Comment (aramperes) [inline README.md]: You can remove this line altogether. The Glowstone project doesn't accept donations at the moment.

File: README.md [hunk 3]
```diff
@@ -51,31 +51,22 @@ If you're using [Maven](https://maven.apache.org/download.html) to manage projec
...
 
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        compile 'com.flowpowered:flow-network:1.0.0'
-    }
+    <repository>
+        <id>glowstone-repo</id>
+        <url>https://repo.glowstone.net/content/repositories/snapshots/</url>
```
Comment (aramperes) [inline README.md]: I made a mistake here, the URL should be `https://repo.glowstone.net/content/repositories/releases/`

File: README.md [hunk 4]
```diff
@@ -51,31 +51,22 @@ If you're using [Maven](https://maven.apache.org/download.html) to manage projec
...
+    </repository>
 
 If you plan on using snapshots and do not already have the snapshot repo in your repository list, you will need to add this as well:
```
Comment (aramperes) [inline README.md]: Remove this paragraph, including the line about manually importing the JAR.

File: README.md [hunk 5]
```diff
@@ -88,5 +79,8 @@ Flow Network is licensed under the [MIT License][License]. Basically, you can do
 * [Spout](https://spout.org/) and contributors - *where we all began, and for much of the re-licensed code.*
 * All the people behind [Java](http://www.oracle.com/technetwork/java/index.html), [Maven](https://maven.apache.org/), and [Gradle](https://www.gradle.org/).
 
-[Donate]: https://flattr.com/submit/auto?user_id=spout&url=https://github.com/flow/network&title=Flow+Network&language=Java&tags=github&category=software
+## Contact
+Feel free to contact us on [discord](https://discord.gg/TFJqhsC)
```
Comment (aramperes) [inline README.md]: `Discord` (with a capital D)

File: README.md [hunk 6]
```diff
@@ -51,31 +51,22 @@ If you're using [Maven](https://maven.apache.org/download.html) to manage projec
     <dependency>
         <groupId>com.flowpowered</groupId>
         <artifactId>flow-network</artifactId>
-        <version>1.0.0</version>
+        <version>1.2.4</version>
     </dependency>
 
-If you're using [Gradle](https://www.gradle.org/) to manage project dependencies, simply include the following in your `build.gradle` file:
+If you're using Maven Project to manage project dependencies, simply include the following in your `pom.xml` file:
```
Comment (aramperes) [inline README.md]: Change this line to

> You will also need to add the Glowstone Maven repository in your `pom.xml` file:

**Review by aramperes — CHANGES_REQUESTED** [review]
Thanks for working on this! I submitted a few more changes we'd like to see. Please make sure you also sign the CLA.

**Comment by preeti98** [comment]
Please do let me know if further changes are required!

**Comment by mastercoms** [comment]
@momothereal Are there any more changes needed?

**Review by aramperes — APPROVED** [review]

**Pull request merged**